code/datums/diseases/advance/symptoms/skin.dm

```diff
@@ -207,10 +207,10 @@ BONUS
 	base_message_chance = 50
 	symptom_delay_min = 60
 	symptom_delay_max = 105
-	var/big_heal
-	var/all_disease
-	var/eggsplosion
-	var/sneaky
+	var/big_heal = FALSE
+	var/all_disease = FALSE
+	var/eggsplosion = FALSE
+	var/sneaky = FALSE
```

Comment:
does this change anything? if not, why change it?

Reply:
Side effect of me really trying to figure out why the thresholds weren't working. I was VVing the symptom and noticed they were null despite the stats supposed to be activating them

Reply:
If you don't initialize them with a value they will be null, which acts like false

Reply:
I am aware it was more a thing of I set a switchboard sorta thing during diagnostics.

Reply:
Doesn't matter I've cleaned that up